### ISSUE TITLE
Type definitions for projects using Typescript

### ISSUE THREAD

**Issue by hkellaway** [launchdarkly/react-native-client-sdk#23]
**Is your feature request related to a problem? Please describe.**
In React Native projects that use Typescript, importing the `launchdarkly-react-native-client-sdk` results in a type error:

```
Could not find a declaration file for module 'launchdarkly-react-native-client-sdk'.... implicitly has an 'any' type.
```

**Describe the solution you'd like**
I'm requesting an official type definition to be included with the LaunchDarkly SDK

**Describe alternatives you've considered**
I searched DefinitelyTyped for a community definition, but none exists.

**Additional context**
N/A

**Comment by torchhound** [comment]
Hi @hkellaway, thanks for suggesting this change! We will put this feature on our internal roadmap.

**Comment by bwoskow-ld** [comment]
Type definitions are now available in 3.0.0.
